decidim-core/app/assets/stylesheets/decidim/modules/_cards.scss

```diff
@@ -440,6 +444,10 @@ $datetime-bg: $primary;
   }
 }
 
+.card--list__text--top{
```

Comment:
If this property-modified always comes along with `.card--list__text`, move it inside that block, and and a `&`. Like:
```scss
.card--list__text {
  ...
  // other properties

  &.card--list__item--top{
     // your properties
  }
}
```